roledb.go

```diff
@@ -495,6 +495,34 @@ func (x *RoleGroupCache) InsertGroup(group *AuthGroup) (err error) {
 	return
 }
 
+func SaveGroup(icentral *Central, group *AuthGroup) bool {
+	if eupdate := icentral.GetRoleGroupDB().UpdateGroup(group); eupdate == nil {
+		return true
+	} else {
+		return false
+	}
+}
+
+func LoadOrCreateGroup(icentral *Central, groupName string, createIfNotExist bool) (*AuthGroup, error) {
```

Comment:
LoadOrCreateGroup should take a RoleGroupDB instead of a Central. Also, place the function directly beneath the definition of the RoleGroupDB interface, since this is a helper function that operates on anything that implements that interface.